cg/cli/demultiplex/finish.py

```diff
@@ -53,43 +46,13 @@ def finish_flow_cell(
 
     flow-cell-name is full flow cell name, e.g. '201203_A00689_0200_AHVKJCDRXX'.
     """
-
-    demux_post_processing_api: DemuxPostProcessingNovaseqAPI = DemuxPostProcessingNovaseqAPI(
-        config=context
-    )
+    demux_post_processing_api: DemuxPostProcessingAPI = DemuxPostProcessingAPI(context)
     demux_post_processing_api.set_dry_run(dry_run)
-    demux_post_processing_api.finish_flow_cell(
-        flow_cell_name=flow_cell_name, force=force, bcl_converter=bcl_converter
-    )
-    # Temporary finish flow cell logic will replace logic above when validated
-    demux_post_processing_api_temp: DemuxPostProcessingAPI = DemuxPostProcessingAPI(config=context)
-    demux_post_processing_api_temp.set_dry_run(dry_run)
-    demux_post_processing_api_temp.finish_flow_cell_temp(
-        flow_cell_directory_name=flow_cell_name, force=force
-    )
-
-
-@finish_group.command(name="temporary")
-@click.argument("flow-cell-directory-name")
-@click.option("--force", is_flag=True)
-@click.pass_obj
-def finish_flow_cell_temporary(context: CGConfig, flow_cell_directory_name: str, force: bool):
-    demux_post_processing_api: DemuxPostProcessingAPI = DemuxPostProcessingAPI(config=context)
     demux_post_processing_api.finish_flow_cell_temp(
-        flow_cell_directory_name=flow_cell_directory_name, force=force
+        flow_cell_directory_name=flow_cell_name, force=force
     )
 
 
-@finish_group.command(name="temporary-all")
-@click.pass_obj
-@DRY_RUN
-def finish_flow_cell_temporary_all(context: CGConfig, dry_run: bool):
-    # Temporary finish flow cell logic will replace logic above when validated
-    demux_post_processing_api_temp: DemuxPostProcessingAPI = DemuxPostProcessingAPI(config=context)
-    demux_post_processing_api_temp.set_dry_run(dry_run=dry_run)
-    demux_post_processing_api_temp.finish_all_flow_cells_temp()
-
-
 @finish_group.command(name="all-hiseq-x")
```

Comment:
We could probably remove this completely, but I think that is better done in a separate PR which we coordinate with updating any automation scripts running this command